spec/takeaway_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'takeaway'
+
+describe Takeaway do
```

Comment:
It looks like your making good use of doubles and stubs here - this means these tests should be fully isolated.

Right now though I don't think your tests for the `Order` class are though.